### PR TITLE
Update README: add setup new project.

### PR DESCRIPTION
**Pull request by icycodes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ git clone https://github.com/TabbyML/quick-question.git
 # Save your OPENAI_API_KEY to file as secret
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
+# Setup your own codebase (Optional).
+# Replace 'your_project_name' and 'your_github_repository_name' with your own,
+# 'your_github_repository_name' could be something like 'TabbyML/quick-question'.
+# See also ./data/diffusers/metadata.json
+mkdir ./data/your_project_name
+echo '{"name":"your_github_repository_name","exampleQueries":["How to ...?"]}' > ./data/your_project_name/metadata.json
```
Comment (icycodes) [inline README.md]: `exampleQueries` is required here.  
Should we set a placeholder in code instead of in metadata.json?

File: README.md [hunk 2]
```diff
@@ -23,6 +23,13 @@ git clone https://github.com/TabbyML/quick-question.git
 # Save your OPENAI_API_KEY to file as secret
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
+# Setup your own codebase (Optional).
```
Comment (wsxiaoys) [inline README.md]: `# Run tabby on your github project`

File: README.md (hunk 1 above)
Comment (wsxiaoys) [inline README.md]: Instead of creating metadata.json with bash, just provide an example file of metadata.json here.

**Review by wsxiaoys — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (icycodes) [reply README.md:26]: Should be "Quick Question"?  
‘Run `tabby` on your github project’ -> 'Run `Quick Question` on your github project'

File: README.md (hunk 1 above)
Comment (wsxiaoys) [reply README.md]: I don't understand.

You can still have exampleQueries in an example with content of `metadata.json` file in markdown.

Comment (icycodes) [reply README.md]: No problem.  
(I asked this question before your review, there's no problem with providing example metadata file.)

File: README.md [hunk 3]
```diff
@@ -17,13 +17,26 @@ An incubating AI-powered Q&A for your codebase.
...
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
-# Start container
+# 3. Run Quick Question on your github project.
+# 3.1. Create a directory in './data' for your project, replace 
+#      'your_project_name' with your own.
+mkdir ./data/your_project_name
+
+# 3.2. Copy metadata template to your project directory.
+cp ./data/metadata_example.json ./data/your_project_name/metadata.json
```
Comment (wsxiaoys) [inline README.md]: Maybe embed the example content here directly. No need to checkin a separate metadata_example.json file.

File: README.md (hunk 2 above)
Comment (wsxiaoys) [reply README.md]: not done.

**Review by wsxiaoys — APPROVED** [review]

**Pull request merged**